tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -84,21 +93,27 @@ def _rename_bucket(cloud_logger, config, source_bucket, source_bucket_details,
         source_bucket_details: The details copied from the source bucket that is being moved
         sts_client: The STS client object to be used
     """
+   
     target_bucket = _create_target_bucket(
         cloud_logger, config, source_bucket_details, config.target_bucket_name)
     sts_account_email = _assign_sts_permissions(cloud_logger, sts_client,
                                                 config, target_bucket)
+
     _run_and_wait_for_sts_job(sts_client, config.target_project,
                               config.bucket_name, config.target_bucket_name,
-                              cloud_logger)
+                              cloud_logger, retain_source_bucket)
+    
+    if retain_source_bucket.lower() == 'true':
```

Comment:
Replace the code as 
if config. retain_source_bucket:
 _print_and_log("'Retaining the source bucket without deleting.")
else:
 _delete_empty_source_bucket(cloud_logger, source_bucket)